tests/visuals/__init__.py

```diff
@@ -0,0 +1,13 @@
+#  Copyright 2022 MTS (Mobile Telesystems)
```

Comment:
Do you know is it correct that we're still using 2022?

Reply:
And btw probably we should add it to every file?

Reply:
I think it works like "since 2022". But I've fixed for newly added file

Reply:
Can you please ask smb about it? I'm sure we have same problems for all new files

Reply:
Ok, waiting for info here